lib/rules/area-alt.json

```diff
@@ -9,7 +9,9 @@
     "wcag412",
     "section508",
     "section508.22.a",
-    "ACT"
+    "ACT",
+    "TTv5",
+    "TT6.a"
```

Comment:
ANDI finds these, correct?

Reply:
Yes 

```html
<map name="infographic">
  <area shape="poly" coords="130,147,200,107,254,219,130,228" href="https://developer.mozilla.org/docs/Web/HTML">
</map>
<img usemap="#infographic" src="https://interactive-examples.mdn.mozilla.net/media/examples/mdn-info2.png" alt="MDN infographic">
```

<img width="996" alt="screenshot of ANDI tool showing accessible name issue for the area element" src="https://user-images.githubusercontent.com/2433219/232525649-a10d8c72-34a6-48ae-94d5-a47089d09d3a.png">